tests/unit/views/SampleReport.spec.js

```diff
@@ -0,0 +1,260 @@
+import { mount } from '@support/testHelper'
+import useRootStore from '@/stores'
+import SampleReport from '@/views/SampleReport.vue'
+import { createPinia, setActivePinia, flushPromises } from '@support/testHelper.js'
+import { fetchFunction } from '@/lib/reports/KinnexReport.js'
+import Reports from '@/lib/reports'
+
+const mockShowAlert = vi.fn()
+
+// We need to hoist and mock the fetch functions to control their behavior in tests
+// and also keep the original functionality of the rest of the module
+// See https://vitest.dev/api/vi.html#vi-mock
+const mocks = vi.hoisted(() => {
+  return {
+    fetchFunction: vi.fn(),
+  }
+})
+
+vi.mock('@/composables/useAlert', () => ({
+  default: () => ({
+    showAlert: mockShowAlert,
+  }),
+}))
+vi.mock('@/lib/csv/creator', async (importOriginal) => {
```

Comment:
needs a suffix

Reply:
Added in [18c2c0d](https://github.com/sanger/traction-ui/pull/2370/commits/18c2c0dad96ea02cebbe6b95e930821d308e7abf)